test/uk/gov/hmrc/testuser/common/SuppressedLogFilter.scala

```diff
@@ -27,11 +26,11 @@ import ch.qos.logback.core.spi.FilterReply
 import play.api.LoggerLike
 
 class SuppressedLogFilter(val messagesContaining: String) extends Filter[ILoggingEvent] {
-  private val suppressedEntries = new mutable.MutableList[ILoggingEvent]()
+  private var suppressedEntries = List[ILoggingEvent]()
```

Comment:
Is immutability really needed here ?

Reply:
MutableList has gone so I replace a mutable val with a var.
Think of anything better?

Reply:
I think there is ArrayBuffer or ListBuffer